src/core/Akka.Tests/IO/TcpIntegrationSpec.cs

```diff
@@ -597,16 +597,16 @@ public void ExpectReceivedData(TestProbe handler, int remaining)
 
             public IPEndPoint Endpoint { get { return _endpoint; } }
 
-            public void Run(Action<TestSetup> action)
+            public async Task RunAsync(Action<TestSetup> action)
```

Comment:
Why do you convert this to async? just delete it, you already have the async version below this.

Reply:
The one below is `RunAsync(Func<TestSetup, Task> asyncAction)`

Reply:
Yes, which is what you need to run an async function, a method that takes a Func that returns a Task